lib/utilities/paginator.rb

```diff
@@ -0,0 +1,85 @@
+module Utilities
+  module Paginator
+    def items_to_display(item:, items_per_page:)
+      @items_per_page = items_per_page || 5
```

Comment:
Keyword arguments can have a default values instead of using this conditional statement